packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -5563,42 +5563,51 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe('#setPermissionTokenPayload', () => {
+        it('sets correctly', () => {
+          assert.notOk(meeting.permissionTokenPayload);
+
+          const permissionTokenPayloadData = {permission: {userPolicies: {a: true}}, exp: '1234'};
+          
+          const jwtDecodeStub = sinon.stub(jwt, 'decode').returns(permissionTokenPayloadData);
```

Comment:
suggestion: This probably doesn't need stubbing - surely we could just set the permission token on the meeting to an actual token value, and let the decode run? I still think it might be worth spying on decode (to ensure it is used), but I think there is value in it actually running.